lib/paynet_api/base_request.rb

```diff
@@ -24,6 +25,10 @@ def base_url
       options[:base_url]
     end
 
+    def proxy_url
```

Comment:
I'm not sure this is setting up the http client to use the proxy. It might be best if we determine a way to build a tests to verify it's using the proxy.